### PR TITLE
Add compat data for color and opacity CSS properties

### PR DESCRIPTION
**Pull request by ddbeck**
This PR migrates compat data for:

* [`caret-color`](https://developer.mozilla.org/docs/Web/CSS/caret-color)
* [`color`](https://developer.mozilla.org/docs/Web/CSS/color)
* [`opacity`](https://developer.mozilla.org/docs/Web/CSS/opacity)

`color` is the big one here, with many features.

### CODE REVIEW

File: css/properties/color.json [hunk 1]
```diff
@@ -0,0 +1,539 @@
+{
+  "css": {
+    "properties": {
+      "color": {
```
Comment (teoli2003) [inline css/properties/color.json]: We need a basic support block.

Comment (ddbeck) [reply css/properties/color.json]: The original table doesn't have a basic support row. What should constitute basic support, in this case? A version that supports one or more of the color values? Supporting all of the CSS level 1 values? Or just assume they're all `true`?

Comment (teoli2003) [reply css/properties/color.json]: All true (1 and 4 for Firefox, Edge: 12, Chrome: 1). Color was in the original css implementations (wasn't version 1 for IE or Opera, but for Chrome, Firefox, and Edge yes.

Comment (ddbeck) [reply css/properties/color.json]: Thank you for these details! I've added the basic support block.

**Pull request merged**